randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -211,20 +215,28 @@ Extra - map_name: rm_a4h03
   > Pickup (Missile Tank)
       All of the following:
           Morph Ball
-          Power Grip or Mid-Air Morph (Intermediate) or Can Use Spider Ball
           Any of the following:
+              # Break the bomb blocks
               Can Use Power Bombs
               All of the following:
                   Can Use Bombs
-                  Infinite Bomb Jumping (Intermediate) or Can Use Spider Ball
+                  Infinite Bomb Jumping (Advanced) or Can Use Spider Ball
```

Comment:
Can you really reach this just with spider ball? seems like you'd need to get to the left wall of the room there and i dont think that's possible without hitting the transition trigger/gliding/HJ/IBJ

Reply:
My thinking was since you had spider you could still climb even if you fell? Not sure how to model this to make it look nice with spider.

Reply:
Can't make it nice, spider's very weird at times :)
just search for the places that require the least amount of reqs to get to a wall, so that you can still arrive at the destination without passing through rooms